corehq/apps/userreports/models.py

```diff
@@ -753,6 +754,52 @@ def rebuild_failed(self):
     def rebuild_awaiting_or_in_progress(self):
         return self.meta.build.awaiting or (self.meta.build.is_rebuild_in_progress and not self.rebuild_failed)
 
+    def set_build_queued(self, *, reset_init_fin=True):
+        self.meta.build.awaiting = True
+        if reset_init_fin:
+            self.meta.build.initiated = None
+            self.meta.build.finished = False
```

Comment:
I noticed this only yesterday when looking at some code,
does this need to also reset initiated_in_place & finished_in_place?
I believe those are just the "in_place" equivalents when "Rebuild in place" is used, and I believe `set_build_queued` will be called for that operation too.

Reply:
Okay, reading more it seems you just retained the implementation as it was.
so, its possible rebuild in place has a bug?

Reply:
Yeah, I didn't change the functionality.

I haven't tested this scenario yet, but I suspect that if there is a bug it wouldn't prevent rebuilds, it would just tell the user than an in-place rebuild is finished when it's actually still in progress. The other possibility is that we don't actually need to reset these values.

I don't think this stuff is covered by unit tests, and I'd like to fix that.